src/tools/compiletest/src/main.rs

```diff
@@ -597,7 +597,7 @@ pub fn is_test(file_name: &OsString) -> bool {
     !invalid_prefixes.iter().any(|p| file_name.starts_with(p))
 }
 
-pub fn make_test(config: &Config, testpaths: &TestPaths) -> test::TestDescAndFn {
+pub fn make_test_with_revisions(config: &Config, testpaths: &TestPaths) -> Vec<test::TestDescAndFn> {
```

Comment:
I don't think we should rename this function. It makes the test with and without revisions. With the new name I kind of expect a "default" `make_test` function somewhere.

Reply:
:+1: will change it back